lib/archethic/db/embedded_impl/chain_reader.ex

```diff
@@ -201,6 +201,21 @@ defmodule Archethic.DB.EmbeddedImpl.ChainReader do
     end
   end
 
+  @doc """
+  Read a transaction from io storage
+  """
+  @spec get_io_transaction(binary(), fields :: list(), db_path :: String.t()) ::
+          Transaction.t() | nil
```

Comment:
As the function `read_transaction` this function should return a tuple `{:ok, tx}` or `{:error, reason}`